client/src/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -52,8 +52,8 @@ const RelatedProducts = ({ productId, handleClick, mainProductDetails, reviews }
           })}
         </ul>
       </div>
-      <a href="#" className="jcarousel-control-prev" data-jcarouselcontrol="true">‹</a>
-      <a href="#" className="jcarousel-control-next" data-jcarouselcontrol="true">›</a>
+      <a href="#" className="jcarousel-control-prev disabled" data-jcarouselcontrol="true">‹</a>
+      <a href="#" className="jcarousel-control-next disabled" data-jcarouselcontrol="true">›</a>
```

Comment:
Would appreciate if you could clarify your reasoning behind this addition. By adding the "disabled" class to these HTML tags, have you permanently disabled/hidden these arrow buttons?

Reply:
I later removed that tag, but I guess I forgot to push that code. That class has/will be removed.